### DISCUSSION TITLE
DevC++ project won't compile

### DISCUSSION THREAD

**Discussion by amosandandy** [community/community#58796] [Programming Help]
### Body

I have this snippet of code that I wanted to talk about on this forum.  When I do so, I usually attach a file with the code example to the post.  Unfortunately, there used to be an Eclipse forum that I have used for many years.  Now they seem to have switched over to GitHub, and I am not familiar with this site!  I was going to attach a file with the pertinent code in it, but, so far, I cannot find any way of doing this when I make a post in this site! 
syd / wt1v

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by airtower-luna** [comment]
> Unfortunately, there used to be an Eclipse forum that I have used for many years. Now they seem to have switched over to GitHub, and I am not familiar with this site!

You might want to check their repository, and the "Discussions" tab there. This forum is for general discussions around Git & GitHub. There might be some Eclipse users around (including myself, on occasion), but you're more likely to get useful answers in their specific place. :wink:

> was going to attach a file with the pertinent code in it, but, so far, I cannot find any way of doing this when I make a post in this site!

Click this bar at the bottom of the field where you write your post/reply:

![grafik](https://github.com/community/community/assets/5814919/43c02703-7144-478e-aca8-3020abec7b1f)
